### ISSUE TITLE
Multi-graphene Chain support

### ISSUE THREAD

**Issue by xeroc** [bitshares/beet#18]
**Is your feature request related to a problem? Please describe.**

Other graphene-based blockchains are facing a similar problem that beet is solving: Trust in third party apps. Some other chains may even experience this as a bigger problem, such as PeerPlays Blockchain which would like to allow 3rd party game developers to develop custom games.

**Describe the solution you'd like**

I'd like beet to be able to not only speak/sign for BitShares, but also other chains (e.g. PeerPlays, SoundDac, Decent, ....).

**Describe alternatives you've considered**

Forking + Customization

**Additional context**

Basically, the app could have a slider of some kind to define which chain you want to talk to. Then, the app would load chain-specific plugins/jslibrary and offer chain-specific API calls.

**Comment by clockworkgr** [comment]
It's in the roadmap already :)

I believe there already is a multi-chain js library for graphene out there.

https://steemit.com/utopian-io/@alexpmorris/howto-encrypt-and-decrypt-bitshares-memos-in-either-javascript-or-java

This article mentions it as a steem-lib fork. Hopefully open sourced soon

**Comment by xeroc** [comment]
Yes, memo stuff is pretty much all the same except for the pubkey prefix.
There even is a JS library for peerplays already in existence that is a fork of bitshares-js:
https://github.com/PBSA/peerplaysjs-lib

**Comment by clockworkgr** [comment]
A lot of preliminary work to support multiple chains done in: https://github.com/bitshares/beet/pull/24

You could already have multiple wallets (each with multiple accounts). Now each wallet is tied to a single (user-selected) chain.

Currently only BTS available...beet-js has been updated accordingly

**Comment by clockworkgr** [comment]
Very close to completion :)

**Comment by sschiessl-bcp** [comment]
https://github.com/bitshares/beet/pull/57#issuecomment-460655045

**Comment by sschiessl-bcp** [comment]
Steem and Forks added, EOS and Forks added (only signed messages)

**Comment by clockworkgr** [comment]
![multi-accounts](https://user-images.githubusercontent.com/6826762/52531294-99403680-2d1b-11e9-99cf-0d9143a5eb23.gif)

**Comment by sschiessl-bcp** [comment]
Supporting now
 - EOS & Forks
 - BitShares & Forks
 - Bitcoin (WTF! :) )
